### PR TITLE
Feature/area of practice

### PR DESCRIPTION
**Pull request by nnagepat**
Feature - Area of practice

### CODE REVIEW

File: Childrens-Social-Care-CPD/Contentful/Renderers/InlineAreaOfPracticeListRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        if (item.Areas.Count == 0)
+        {
+            return NoAreasOfpractice();
+        }
+
+        var htmlContentBuilder = new HtmlContentBuilder();
+
+        foreach (var contentItem in item.Areas)
+        {
+            var areaOfpractice = contentItem.Items.Find(x => typeof(AreaOfPractice) == x.GetType()) as AreaOfPractice;
```
Comment (cairnsj) [inline Childrens-Social-Care-CPD/Contentful/Renderers/InlineAreaOfPracticeListRenderer.cs:27]: You can use OfType<T> here.

File: Childrens-Social-Care-CPD/Contentful/EntityResolver.cs
Comment (cairnsj) [inline Childrens-Social-Care-CPD/Contentful/EntityResolver.cs:1]: Just to note that these are ordered alphabetically

File: Childrens-Social-Care-CPD/Contentful/PartialsFactory.cs
Comment (cairnsj) [inline Childrens-Social-Care-CPD/Contentful/PartialsFactory.cs:1]: Just to note that these are ordered alphabetically

File: browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts
Comment (cairnsj) [inline browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts:1]: Is this file needed?

**Review by cairnsj — CHANGES_REQUESTED** [review]
Remove the empty file if not needed.

File: browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts
Comment (nnagepat) [reply browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts:1]: Its a place holder for any future tests when user adds any content with external links

File: Childrens-Social-Care-CPD/Contentful/EntityResolver.cs
Comment (nnagepat) [reply Childrens-Social-Care-CPD/Contentful/EntityResolver.cs:1]: Resolved

File: Childrens-Social-Care-CPD/Contentful/PartialsFactory.cs
Comment (nnagepat) [reply Childrens-Social-Care-CPD/Contentful/PartialsFactory.cs:1]: Resolved

File: browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts
Comment (nnagepat) [reply browser-tests/content-regression-tests/tests/explore-area-of-practice/external-links.spec.ts:1]: Resolved

**Review by cairnsj — APPROVED** [review]
Looks good
